src/Storage/Commands.Storage/help/Get-AzureStorageBlob.md

```diff
@@ -14,14 +15,14 @@ Lists blobs in a container.
 
 ### BlobName (Default)
 ```
-Get-AzureStorageBlob [[-Blob] <String>] [-Container] <String> [-MaxCount <Int32>]
+Get-AzureStorageBlob [[-Blob] <String>] [-Container] <String> [-IncludeDeleted] [-MaxCount <Int32>]
```

Comment:
Please include output in the examples (the updated output)

Reply:
The output added to example. Please check.